assets/account.css

```
@@ -227,16 +242,121 @@
   background: var(--background-accent);
 }
 
-.account__error-message {
-  font-size: 14px;
+.account-checkbox__label--error:has(+ .account__error-message:not(:empty)) {
+  color: var(--color-red);
+}
+
+.account-checkbox__label--error:has(+ .account__error-message:not(:empty)):before {
+  border-color: var(--color-red);
+  background: var(--color-red-light);
+}
+
+.account-checkbox__label--error:has(+ .account__error-message:not(:empty)):after {
+  border-color: var(--color-red);
+}
+
+.account-checkbox__label--error:has(+ .account__error-message:not(:empty)) .account__agreement-inner  {
+  color: var(--color-primary);
+}
+
+.account__agreement:after {
+  content: "";
+  position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  background: var(--color-primary-67);
+  opacity: 0;
+  visibility: hidden;
+}
+
+.account__agreement:has([id^='user_agreement_opener']:checked):after {
+  opacity: 1;
+  visibility: visible;
+}
+
+.account__agreement-modal {
+  display: block !important;
+  position: fixed;
+  top: calc(var(--header-height, 200px) + var(--header-transform, 0px));
+  left: 0;
+  width: 100%;
+  height: calc(100vh - var(--header-height, 200px) - var(--header-transform, 0px));
+  transition: all var(--animation-duration) var(--transition-function-ease-out);
+  padding: 10px;
+  visibility: hidden;
+  opacity: 0;
+  z-index: 1;
+}
+
+.account__agreement-content {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  max-width: calc(var(--max-width-narrow) + 60px);
+  padding: 20px;
+  margin: 0 auto;
+  display: block;
+}
+
+.account__agreement-inner {
+  border: 1px solid var(--color-border);
+  width: 100%;
+  height: 100%;
+  display: block;
+  overflow: hidden;
+  border-radius: var(--border-radius-rounded-blocks);
+  background: var(--background-primary);
+  color: var(--color-primary);
+}
+
+.account__agreement-text {
+  display: block;
+  padding: 24px;
+  height: 100%;
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+.account__agreement-closer {
+  position: absolute;
+  top: 0;
+  right: 6px;
+  width: 40px;
+  height: 40px;
+  text-indent: -9999px;
+  overflow: hidden;
+  cursor: pointer;
+  transition: all var(--animation-duration) var(--transition-function-ease-out);
+}
+
+.account__agreement-closer:after,
+.account__agreement-closer:before {
+  content: "";
   position: absolute;
-  right: 12px;
   top: 50%;
-  transform: translate(0, -50%);
-  padding: 0 5px;
-  background:  var(--color-red-light);
-  color: var(--color-red);
-  pointer-events: none;
+  left: 23%;
+  right: 23%;
+  transform: rotate(45deg);
+  height: 2px;
+  background: var(--color-accent);
```

I didn't find where the widths for the pseudo elements were set, so please add it too
```suggestion
  background: var(--color-accent);
  width: 20px;
```

---

`width` prop is not necessary if there are both `left` and `right` props and on the other hand `width` and `left` or `width` and `right`.
This time I used an approach with
```
top: 50%;
left: 50%;
transform: translate(-50%, -50%) rotate(45deg);
```
and noticed it worked not entirely accurately earlier because some styles didn't exist on its parent element but everything is good now